crates/cheatnet/src/execution/contract_execution_syscall_handler.rs

```diff
@@ -137,3 +129,22 @@ impl ResourceTracker for ContractExecutionSyscallHandler<'_, '_> {
             .run_resources()
     }
 }
+
+fn as_printable_short_string(value: &Felt252) -> Option<String> {
+    let bytes: Vec<u8> = value.to_bytes_be();
+    if bytes.iter().any(u8::is_ascii_control) {
+        return None;
+    }
+
+    as_cairo_short_string(value)
+}
```

Comment:
(Knowledge sharing, no need to change though would make code more idiomatic, as from traits should be used for conversions)
We could create a type
```
struct PrintableString(String);
```
and implement it as a trait
```
From<Felt252> impl PrintableString
```